examples/catch22.ipynb

```diff
@@ -30,7 +30,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
```

Comment:
updating hte notebook is fine, just make sure you have reset and run it only once, in the final commit.

Reply:
got it, I rerun and committed now